src/Pickles/Pickles/DocumentationBuilders/HTML/HtmlStepFormatter.cs

```diff
@@ -19,8 +19,11 @@
 //  --------------------------------------------------------------------------------------------------------------------
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
 using System.Xml.Linq;
-
+using DocumentFormat.OpenXml.Office.Word;
```

Comment:
Please remove this unused using - especially since Word and Html are different things :-)